tests/performance/performancetests.sh

```diff
@@ -383,6 +452,27 @@ if [ -z "$EXPTYPE" ] || ! [ "$EXPTYPE" == "KERNEL" ]; then
 			fi
 		done
 
+		for ((i=0;i<${#MULTIPLICATION_DATASETS[@]};++i));
+		do
+			if [ ! -z "$DATASETTORUN" ] && [ "$DATASETTORUN" != "${MULTIPLICATION_DATASETS[i]}" ]; then
+				continue
+			fi
+
+			# initialise parameters
+			DATASET=${MULTIPLICATION_DATASETS[i]}
+			PARSE_MODE=${MULTIPLICATION_DATASET_MODES[i]}
+
+			# test for file
+			if [ ! -f ${INPUT_DIR}/${DATASET} ]; then
+				echo "Warning: dataset/${DATASET} not found. Provide the dataset to enable performance tests with it."
```

Comment:
Please include the phrase `Test DISABLED`, probably instead of `Warning`, so that `./tests/summarise.sh` picks up on it